metaflow/plugins/kubernetes/kubernetes.py

```diff
@@ -149,6 +149,10 @@ def create_job(
         gpu_vendor=None,
         disk=None,
         memory=None,
+        use_tmpfs=None,
```

Comment:
I should've asked this earlier. Why do we need this config option? If the other `tmpfs` config options are specified, should't it automatically mean that `use_tmpfs` is True?

Reply:
I think I know why.. The simplest way to run this would be to simply use `--use-tmpfs`. The other config otpions have default values that will get used if tmpfs is enabled. Is that correct?

Reply:
correct, and it's the way the batch tmpfs feature works as well.